aws-distro-opentelemetry-node-autoinstrumentation/src/aws-opentelemetry-configurator.ts

```diff
@@ -943,7 +943,12 @@ export function isLambdaEnvironment() {
 }
 
 function hasCustomOtlpTraceEndpoint() {
```

Comment:
nit: Could you help update the comment at line: 718? 
Overall the PR looks good